scripts/aws_bastion_mfa_export.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
File permissions are set to 644 should we change to 755 so its ready to go when cloned?

Reply:
yeah, this sounds good to me!

I changed the file permissions: https://github.com/puppetlabs/puppetlabs-autope/pull/35/commits/cb736f4cb607884e0fbe3a8d071b98d8b329b42e